services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/hakemus/HakemusMigrationService.kt

```diff
@@ -28,19 +31,48 @@ class HakemusMigrationService(
     fun migrateOneHanke(hankeId: Int) {
         val hanke = hankeRepository.getReferenceById(hankeId)
 
-        hanke.hakemukset.singleOrNull()?.let {
-            val founder = createFounderKayttaja(hanke.id, it.applicationData)
-            val otherKayttajat = createOtherKayttajat(it.applicationData, founder, hanke.id)
-            createYhteystiedot(it, (otherKayttajat + founder).filterNotNull())
-            it.applicationData = clearCustomers(it.applicationData)
-        }
+        hanke.hakemukset.singleOrNull()?.let { hakemus: ApplicationEntity ->
+            logger.info { "Migrating hakemus. ${hakemus.logString()}, ${hanke.logString()}" }
+
+            val existingKayttajat = hankekayttajaRepository.findByHankeId(hanke.id)
+            logger.info {
+                if (existingKayttajat.isNotEmpty()) {
+                    "Hanke already has users. ${existingKayttajat.size} users with IDs: " +
```

Comment:
Unnecessary repetition? Maybe just `Hanke already has ${existingKayttajat.size} users with IDS: ...`

Reply:
I tried to keep the log entries searchable. So this can be filtered by the message "Hanke already has users."

Reply:
Ok